<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>195</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="Diagnostic_Tab2_SettingsGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mohn's Rho&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Mohn's Rho is a metric used to describe the degree of retrospectivity (i.e. a retrospective pattern) for a particular variable. It takes an average of the relative differences of an estimate from a truncated time series assessment and an estimate from a full time series assessment. The truncated time series assessments are referred to as &amp;quot;peels&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>Mohn's Rho Settings:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="Diagnostic_Tab2_NumPeelsLBL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>The number of years removed from the time series</string>
          </property>
          <property name="statusTip">
           <string>The number of years removed from the time series</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Years Peeled:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Diagnostic_Tab2_NumPeelsSB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>The number of years removed from the time series</string>
          </property>
          <property name="statusTip">
           <string>The number of years removed from the time series</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Diagnostic_Tab2_ConvOnlyCB">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Allow only converged runs</string>
          </property>
          <property name="statusTip">
           <string>Allow only converged runs</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Allow Converged Runs&lt;/span&gt;&lt;/p&gt;&lt;p&gt;A retrospective analysis involves as many runs as years peeled selected by the user. If any of these runs shouldn't converge, then checking this box could result in a model running forever. Therefore, it's recommended that this box only be checked if the user is sure all the retrospective analysis runs will converge.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Conv Only:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Diagnostic_Tab2_PeelPositionLT">
        <item>
         <widget class="QLabel" name="Diagnostic_Tab2_PeelPositionLBL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>The position in the time series where the truncation takes place</string>
          </property>
          <property name="statusTip">
           <string>The position in the time series where the truncation takes place</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Peel Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Diagnostic_Tab2_PeelPositionCMB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>The position in the time series where the truncation takes place</string>
          </property>
          <property name="statusTip">
           <string>The position in the time series where the truncation takes place</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <item>
           <property name="text">
            <string>From End</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>From Beginning</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="Diagnostic_Tab2_YearRangeLBL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>The year range as defined by the above Peel data</string>
          </property>
          <property name="statusTip">
           <string>The year range as defined by the above Peel data</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Year Range:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Diagnostic_Tab2_MinYearLBL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>First year of time series</string>
          </property>
          <property name="statusTip">
           <string>First year of time series</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>2000</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Diagnostic_Tab2_MinYearLE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>First year of peel</string>
          </property>
          <property name="statusTip">
           <string>First year of peel</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Diagnostic_Tab2_YearRangeLBL_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>to</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Diagnostic_Tab2_MaxYearLE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Last year of peel</string>
          </property>
          <property name="statusTip">
           <string>Last year of peel</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Diagnostic_Tab2_MaxYearLBL">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Last year of time series</string>
          </property>
          <property name="statusTip">
           <string>Last year of time series</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Peeled Years&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Peeled years refer to the number of years removed from the time series. For example, if the time series is 20 years long, starting with the year 2000, and the user selects 3 years to be peeled from the end, Mohn's Rho will be calculated using the time ranges [2000,2017], [2000,2018], and [2000,2019].&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>2000</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_84">
     <item>
      <spacer name="horizontalSpacer_96">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Diagnostic_Tab2_RunPB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Run Mohn's Rho Analysis</string>
       </property>
       <property name="statusTip">
        <string>Run Mohn's Rho Analysis</string>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_97">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
